tools/config-generator/ConfigurationGenerator.cpp

```diff
@@ -0,0 +1,118 @@
+#include "vara/Feature/FeatureModel.h"
+#include "vara/Sampling/SampleSetParser.h"
+#include "vara/Sampling/SampleSetWriter.h"
+#include "vara/Solver/ConfigurationFactory.h"
+
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Errc.h"
+#include "llvm/Support/InitLLVM.h"
+#include "llvm/Support/MemoryBuffer.h"
+
+static llvm::cl::OptionCategory
+    ConfigCreatorCategory("Configuration generator options");
+
+static llvm::cl::list<std::string>
+    FileNames(llvm::cl::Positional, llvm::cl::desc("Path to the feature model"),
```

Comment:
Why allow multiple file names if we only process one? We could just change this to a singe filename or?

Reply:
Yep. Changed it.